cmd/kubeadm/app/cmd/token.go

```diff
@@ -277,22 +295,82 @@ func RunListTokens(out io.Writer, errW io.Writer, client clientset.Interface) er
 		return errors.Wrap(err, "failed to list bootstrap tokens")
 	}
 
-	w := tabwriter.NewWriter(out, 10, 4, 3, ' ', 0)
-	fmt.Fprintln(w, "TOKEN\tTTL\tEXPIRES\tUSAGES\tDESCRIPTION\tEXTRA GROUPS")
-	for _, secret := range secrets.Items {
+	if outputFormat == "text" {
```

Comment:
Can we have this logic into another util function? It will be more modular and easy for testing.

Reply:
^ which is a good reminder that we might want to add unit tests under _token.go